frontend/src/styles.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
Might want to wait to commit empty files until they have something in them as this is just noise.

Reply:
Ah..ok